src/mainboard/lenovo/thinkcentre_m700/mainboard.c

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include <device/device.h>
+#include <drivers/intel/gma/int15.h>
+
+static void mainboard_enable(struct device *dev)
+{
+	install_intel_vga_int15_handler(GMA_INT15_ACTIVE_LFP_NONE,
```

Comment:
That applies to LCD panels via LVDS interface. Normal VGA or DP should work without it.